pkg/server/datastore/sqlstore/sqlstore_test.go

```diff
@@ -3693,6 +3693,52 @@ func (s *PluginSuite) TestDeleteBundleDissociateRegistrationEntries() {
 	s.Require().Empty(entry.FederatesWith)
 }
 
+func (s *PluginSuite) TestListRegistrationEntriesEvents() {
+	entry1 := s.createRegistrationEntry(&common.RegistrationEntry{
+		Selectors: []*common.Selector{
+			{Type: "Type1", Value: "Value1"},
+		},
+		SpiffeId: "spiffe://example.org/foo1",
+		ParentId: "spiffe://example.org/bar",
+	})
+	resp, err := s.ds.ListRegistrationEntriesEvents(ctx, &datastore.ListRegistrationEntriesEventsRequest{})
+	s.Require().NoError(err)
+	s.Require().Equal(1, len(resp.EntryIDs))
```

Comment:
may we compare the expected IDs? tehe same comment is for all equal compartions
Using a list of expected

Reply:
Added comparisons